src/native/corehost/fxr/install_info.h

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef __INSTALL_INFO_H__
+#define __INSTALL_INFO_H__
+
+#include "pal.h"
+
+namespace install_info
+{
+    bool print_environment(const pal::char_t* leading_whitespace);
+    bool print_installs(const pal::char_t* leading_whitespace, bool skip_current_arch);
```

Comment:
I assume the `skip_current_arch` is in preparation for pritning this in error messages as well? Currently it seems to always be set to `false`.

Reply:
Ah, I was trying out showing all versus just showing other architectures. It is always skipping the current arch right now (which I think would be reasonble for error messages in the future too). Will remove.